internal/mode/static/nginx/config/servers_test.go

```diff
@@ -827,7 +890,10 @@ func TestCreateServers(t *testing.T) {
 
 	g := NewWithT(t)
 
-	result := createServers(httpServers, sslServers)
+	result, httpMatchPair := createServers(httpServers, sslServers)
+	if httpMatchPair != nil {
```

Comment:
this seems like a bug. what if returned `httpMatchPair ` is nil. In that buggy case, the test will succeed.